docs/CBridgeFacet.md

```diff
@@ -53,7 +41,7 @@ The swap library can be found [here](../src/Libraries/LibSwap.sol).
 
 ## LiFi Data
```

Comment:
This is present in all facets, should this be ## BridgeData? 
Also the text below says "some methods" where in fact we always pass the bridge data as param?